docs/StepbyStepManualDeploymentGuideScalarDLOnAzure.md

```diff
@@ -0,0 +1,237 @@
+# Example steps to Deploy Scalar DL on Azure
+
+This guide assumes that you create the environment on the basis of [manual deployment guide for Azure](./ManualDeploymentGuideScalarDLOnAzure.md).
+This document contains example steps for creating a Scalar DL environment in the Azure cloud.
+
+## Prerequisites
+
+You can skip the following steps If the user has an [Owner](https://docs.microsoft.com/en-us/azure/role-based-access-control/built-in-roles#owner) or [Contributor](https://docs.microsoft.com/en-us/azure/role-based-access-control/built-in-roles#contributor) role in Azure.
+You can use the following steps to create and assign a custom role to the user. User should have `Privileged Role Administrator` or `Global Administrator` to do the following
+
+* Select [Subscriptions](https://portal.azure.com/#blade/Microsoft_Azure_Billing/SubscriptionsBlade) from Azure portal
+* Select **Subscription name**
+* Select **Access control (IAM)**
+* Click **Add**
+* Select **Add custom role**
+    * On the Basics page, configure the following options
+        * Enter a **custom role name**
+        * Enter a **Description**
+        * Select **Start from scratch**
+        * Click **Next**
+    * On the Assignable scopes page, configure the following options
+        * Select your **Subscription**
+        * Click **Next**
+    * On the JSON page
+        * Click **Edit**
+        * Paste the permissions section from [JSON](./CloudPermissionsForScalarDLOnAKS.md)
+        * Click **Next**
+    * On **Review+Create** page
+        * Click **Create**
+        * Click **OK**
+* On the Access control IAM page in subscription, configure the following options
+    * Click **Add**
+    * Click **Add role assignment**, enter the following details in the blade that can be seen on the right side
+        * Select created custom role from **Role**
+        * Select User, group or service principal from **Assign access to**
+        * Select desired username from **Select**
+        * Click **Save**
+
+Add _Directory reader_ permission for the user to access the service principal
+* Select [Azure Active Directory](https://aad.portal.azure.com/) from Azure portal
```

Comment:
It maybe better to use this URL.

```suggestion
* Select [Azure Active Directory](https://aad.portal.azure.com/#blade/Microsoft_AAD_IAM/ActiveDirectoryMenuBlade/Overview) from Azure portal
```

Reply:
Thank you for the suggestion, updated in aec4f67